extensions/amp-story/1.0/amp-story-store-service.js

```diff
@@ -565,6 +567,7 @@ export class AmpStoryStoreService {
           [StateProperty.CAN_INSERT_AUTOMATIC_AD]: false,
           [StateProperty.CAN_SHOW_BOOKEND]: false,
           [StateProperty.CAN_SHOW_NAVIGATION_OVERLAY_HINT]: false,
+          [StateProperty.CAN_SHOW_PAGINATION_BUTTONS]: false,
```

Comment:
Can you add a default that'd be `true`? (sorry, can't comment on the actual line where this would happen...) Otherwise the pagination buttons would never be visible

Reply:
Whoops, done